examples_tests/61.BlitFilterTest/normalization/normalization.glsl

```diff
@@ -0,0 +1,92 @@
+#ifndef _NBL_GLSL_BLIT_NORMALIZATION_INCLUDED_
+#define _NBL_GLSL_BLIT_NORMALIZATION_INCLUDED_
+
+#include <nbl/builtin/glsl/workgroup/arithmetic.glsl>
+
+uint integerDivide_64_32_32(in uint dividendMsb, in uint dividendLsb, in uint divisor)
+{
+	const uint MAX_UINT = ~0u;
+	const float msbRatio = float(dividendMsb) / float(divisor);
+	const uint quotient = uint((msbRatio * MAX_UINT) + msbRatio) + dividendLsb / divisor;
+	return quotient;
+}
+
+#ifndef _NBL_GLSL_BLIT_NORMALIZATION_MAIN_DEFINED_
+
+#ifndef _NBL_GLSL_SCRATCH_SHARED_DEFINED_
+#error "_NBL_GLSL_SCRATCH_SHARED_DEFINED_ must be defined"
+#endif
+
+#define scratchShared _NBL_GLSL_SCRATCH_SHARED_DEFINED_
```

Comment:
usually if you dont declare it, `#include <nbl/builtin/glsl/workgroup/arithmetic.glsl>` will declare it on its own.